apps/client/src/app/map/page.tsx

```
@@ -0,0 +1,13 @@
+"use client";
+
+import KakaoMap from "@/components/KakaoMap";
+import { Button } from "@place-memo/ui";
+
+export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)